system/jhtree/jhtree.cpp

```diff
@@ -2633,6 +2682,7 @@ const CJHTreeNode *CNodeCache::getNode(const INodeLoader *keyIndex, unsigned iD,
             cacheEntry = new CNodeCacheEntry;
             curCache.replace(key, *cacheEntry);
             alreadyExists = false;
+            curCache.numAdds->fastInc(1);
```

Comment:
This can be within the replace() function

Reply:
Ditto...